packages/icu/SOURCES/exclude-burmese.patch

```diff
@@ -1,8 +1,10 @@
 --- icu.orig/source/data/BUILDRULES.py       2020-05-24 19:06:58.842238240 +0000
 +++ icu/source/data/BUILDRULES.py            2020-05-24 17:38:47.929485280 +0000
-@@ -221,7 +221,6 @@
+@@ -220,8 +220,8 @@
+     # Dict Files
      input_files = [InFile(filename) for filename in io.glob("brkitr/dictionaries/*.txt")]
      output_files = [OutFile("brkitr/%s.dict" % v.filename[20:-4]) for v in input_files]
++    output_files = filter(lambda f: 'burmese' not in f, output_files)
```

Comment:
If the `dat` file approach works (it still is building and is very slow for me on my Octane), we can drop the other patches!